src/test/pivot_root.c

```diff
@@ -0,0 +1,41 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "nsutils.h"
+#include "rrutil.h"
+
+int main(void) {
+  try_setup_ns(CLONE_NEWNS);
```

Comment:
Yes, it should thanks. I considered having this be inside the implementation, but then it seemed better for it to not be quite that magic and have the skipping be quite explicit.